cns/service/main.go

```diff
@@ -813,6 +813,11 @@ func main() {
 		if platform.HasMellanoxAdapter() {
 			go platform.MonitorAndSetMellanoxRegKeyPriorityVLANTag(rootCtx, cnsconfig.MellanoxMonitorIntervalSecs)
 		}
+
+		// if swiftv2 scenario is enabled, we need to initialize the Service Fabric (standalone) swiftv2 middleware to process IP configs requests
+		// we don't need a config check here since this middleware will be called only for swiftv2 path for standalone
```

Comment:
Please mention here that it is CNI that will call this middleware for Swift v2 path. CNI has a swiftv2 flag.
For AKS swiftv2, different handler gets called.